debian/control

```diff
@@ -41,7 +41,7 @@ Architecture: any
 Depends: adduser,
          ca-certificates,
          gnupg1 | gnupg,
-         snap-confine (>= 1.0.43),
+         snap-confine (>= 2.19),
```

Comment:
A tiny tweak we could do here is to make it `>= 2.19~` to make it backport friendly (mostly for debian).